src/main/java/net/dv8tion/jda/api/managers/ChannelManager.java

```diff
@@ -467,4 +471,25 @@ default ChannelManager sync()
     @Nonnull
     @CheckReturnValue
     ChannelManager setBitrate(int bitrate);
+
+    /**
+     * Sets the <b><u>news flag</u></b> of the selected {@link net.dv8tion.jda.api.entities.TextChannel TextChannel}.
+     *
+     * @param  news
+     *         The new news flag for the selected {@link net.dv8tion.jda.api.entities.TextChannel TextChannel},
+     *
+     * @throws net.dv8tion.jda.api.exceptions.InsufficientPermissionException
+     *         If the currently logged in account does not have the Permission {@link net.dv8tion.jda.api.Permission#MANAGE_CHANNEL MANAGE_CHANNEL}
+     * @throws IllegalStateException
```

Comment:
Why have here the short and below the full when it's the same thing?
Also, wouldn't it be better to list both under one `@throws`?

Reply:
that's not constant in other docs too